### DISCUSSION TITLE
Taking into account the plural in translation

### DISCUSSION THREAD

**Discussion by jean-louis67** [photoprism/photoprism#4674] [General]
Sorry if this question is already mentioned in a discussion.
To translate correctly in French: _{0} appended action_ which is invariable in English, it takes two forms: 

- _{0} action attachée_ if the number is equal to 0 or 1
- _{0} actions attachées_ if the number is greater than 1.

I have not understood how to make this differentiation.

A degraded form can then be: _{0} actions(s) attachée(s)_
Thanks in advance for your opinion.

**Comment by graciousgrey** [comment]
Thank you very much for improving the translations!

Unfortunately it is not yet possible to provide a singular and a plural form so I think _{0} actions(s) attachée(s)_ is a good solution until we support plural strings.
